web/app/test/testHelpers.jsx

```diff
@@ -0,0 +1,21 @@
+import _ from 'lodash';
+import { createMemoryHistory } from 'history';
+import React from 'react';
+import { Route, Router } from 'react-router';
+
+export function printStack() {
```

Comment:
I would mayyyybe remove printStack() since it's kind of a debugging helper anyway, and that console.log will keep us from passing lint.

Reply:
ok, removing that function 😄